packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -40,6 +51,10 @@ export function getBeaconPoolApi({
       return {data: chain.opPool.getAllAttesterSlashings()};
     },
 
+    async getPoolAttesterSlashingsV2() {
```

Comment:
We need to decide about how we wanna handle the fork boundary as op apis can and should only return one attestation type (phase0 or electra), I had a quick chat with @twoeths about this.

Possible solutions are
- prune op pools at fork transition, all pre-electra messages become useless as they can not be included in a block anymore
- fitler out pre-electra messages before returning result on api

Pruning seems like a good solution to me, but need to see if it's worth it since those messages would be pruned out over time anyways.